Woof/Woof/View/OwnerFlow/OwnerProfileView/OwnerProfileView.swift

```diff
@@ -9,39 +9,84 @@ struct OwnerProfileView: View {
     @State private var isEditingMode = false
 
     var body: some View {
-        VStack {
-            if isEditingMode {
-                EditOwnerInformationView(
-                    name: $viewModel.name,
-                    surname: $viewModel.surname,
-                    phone: $viewModel.phone,
-                    address: $viewModel.address
-                )
-            } else {
-                OwnerCardView(
-                    name: viewModel.name,
-                    surname: viewModel.surname,
-                    phone: viewModel.phone,
-                    address: viewModel.address,
-                    avatarUrl: viewModel.avatarURL
-                )
-            }
-            Spacer()
-        }
-        .padding(.horizontal)
-        .overlay(alignment: .topTrailing) {
-            Button(isEditingMode ? "Save" : "Edit") {
+        NavigationView {
+            VStack {
                 if isEditingMode {
-                    viewModel.save()
+                    VStack {
+                        EditOwnerInformationView(
+                            name: $viewModel.name,
+                            surname: $viewModel.surname,
+                            phone: $viewModel.phone,
+                            address: $viewModel.address
+                        )
+
+                        Button(isEditingMode ? saveButtonLabelText : editButtonLabelText) {
+                            if isEditingMode {
+                                viewModel.save()
+                            }
+                            isEditingMode.toggle()
+                        }
+                    }
+                    .padding()
+                    .buttonStyle(CapsuleWithWhiteText())
+                    .background(Color.App.purpleLight)
+                    .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
+                } else {
+                    VStack {
+                        OwnerCardView(
+                            name: viewModel.name,
+                            surname: viewModel.surname,
+                            phone: viewModel.phone,
+                            address: viewModel.address,
+                            avatarUrl: viewModel.avatarURL
+                        )
+
+                        Button(isEditingMode ? saveButtonLabelText : editButtonLabelText) {
+                            if isEditingMode {
+                                viewModel.save()
+                            }
+                            isEditingMode.toggle()
+                        }
```

Comment:
What changes if to do in this way? 
```suggestion
                        Button(editButtonLabelText) {
                            isEditingMode.toggle()
                        }
```